modules/base_infra/vpc_infra/vpc_subnets.tf

```diff
@@ -1,36 +1,15 @@
-resource "aws_vpc" "filmdrop_vpc" {
-  cidr_block           = var.vpc_cidr
-  enable_dns_hostnames = true
-  enable_dns_support   = true
-
-  tags = {
-    Name = "${local.name_prefix}-vpc"
-  }
+data "aws_vpc" "filmdrop_vpc" {
+  cidr_block = var.vpc_cidr
 }
 
-resource "aws_subnet" "public_subnets" {
+data "aws_subnet" "public_subnets" {
   for_each = var.public_subnets_az_to_id_map
-
-  vpc_id            = aws_vpc.filmdrop_vpc.id
-  cidr_block        = each.value
-  availability_zone = each.key
```

Comment:
Can you run a `pre-commit run --all-files`

Reply:
done - pushed tflint fix